app/com/linkedin/drelephant/exceptions/AzkabanClient.java

```diff
@@ -0,0 +1,207 @@
+package com.linkedin.drelephant.exceptions;
+
+import controllers.HtmlUtil;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.conn.scheme.Scheme;
+import org.apache.http.conn.ssl.SSLSocketFactory;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.log4j.Logger;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.NameValuePair;
+
+public class AzkabanClient {
+
+    private final Logger logger = Logger.getLogger(AzkabanClient.class);
+    private String azkabanUrl;
+    private String executionId;
+    private String sessionId;
+
+    public AzkabanClient(String url) {
+        if (azkabanUrl == null || azkabanUrl.isEmpty()) {
+            logger.info("Empty azkaban url");
+        }
+        this.azkabanUrl = url;
+        try{
+            this.executionId = new URL(url).getQuery().toString().substring(7);
```

Comment:
Use constant variable for 7. Give a meaning ful name to explain it's significance.